Rubberduck.VBEEditor/Application/HostApplicationBase.cs

```diff
@@ -68,6 +119,73 @@ public virtual object Run(string name, params object[] args)
             return null;
         }
 
+        private int? _rcwReferenceCount;
```

Comment:
Seems to me that the `ApplicationHostBase` really should inherit from `SafeComWrapper` so we wouldn't have to copy all that code here?

Reply:
I agree. It really is a wrapper for a COM object.
Any concerns @retailcoder ?